lib/rmt/cli/smt_importer.rb

```diff
@@ -3,19 +3,21 @@
 
 
 class SMTImporter
-  attr_accessor :config
+  attr_accessor :data_dir
+  attr_accessor :no_systems
 
   class ImportException < StandardError
   end
 
-  def initialize(config)
```

Comment:
`def initialize(data_dir, no_systems)`

I don't think that works? You're still passing a single `config` from `bin/rmt-data-import`

I was thinking this would be like:

```ruby
def initialize(config)
  @data_dir = config.data_dir
  @no_systems = config.no_systems
end
```